src/hyperfine/error.rs

```diff
@@ -40,6 +41,13 @@ impl fmt::Display for ParameterScanError {
                  floating point numbers. The step size can be specified \
                  with the '--parameter-step-size' parameter"
             ),
+            ParameterScanError::DifferentCommandNameCountWithParameters(real, expected) => {
+                write!(
+                    f,
+                    "Current --command-name count is {}: expected {} as parameters",
```

Comment:
Since `OptionsError::UnexpectedCommandNameCount` is returned from [code]( https://github.com/sharkdp/hyperfine/pull/391/files#diff-42cb6807ad74b3e201c5a7ca98b911c5fa08380e942be6e4ac5807f8377f87fcR263) for the command `--parameter-list`.

```
hyperfine 'echo {foo}' --parameter-list foo 1,2 --command-name name-{foo}
```

And `ParameterScanError::UnexpectedCommandNameCount` is returned from [code]( https://github.com/sharkdp/hyperfine/pull/391/files#diff-29a999f73979d97d17efda3664084ed0d2ea9d1a05e8bc5f032f94136c4f0e43R109) for the command `--parameter-scan`.

```
hyperfine 'echo {val}' --parameter-scan val 1 3 --parameter-step-size 1 --command-name name-{val}
```

Reply:
Thanks!